intrinsic_sdk_cmake/cmake/sdk_version.json

```diff
@@ -1,3 +1,4 @@
 {
-	"sdk_version": "v1.17.20250331"
+	"sdk_version": "v1.19.20250519",
+	"sdk_checksum": "SHA256=91cea6ba18fdd67809ecbc5adf17d250e9a6652743e3945b7ddd94c00460b9a8"
```

Comment:
Long term it might make sense to change this to just be the hash and/or split the hash type out as a separate field, depending on how other tools use it. But I think it's good for now.